fashion-messaging-app-ui/src/App.jsx

```diff
@@ -53,6 +56,11 @@ export default function App() {
     setSearchQuery(query);
   };
 
+
+  useEffect(() => {
+    localStorage.setItem('user', JSON.stringify(user));
+  }, [user]);
```

Comment:
This stores the user information in local storage

Reply:
Do you need to store the user in local storage if you can get the user from useContext(UserContext) ?